auth_token/contrib/is_core_auth/resource.py

```diff
@@ -1,5 +1,6 @@
-try:
-    import security
+from auth_token.utils import is_installed_security
+
+if is_installed_security:
     from .auth_security.resource import AuthResource
-except ImportError:
```

Comment:
wrong idea. Because in from .auth_security.resource import AuthResource can be import error